src/Microsoft.AspNet.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -97,9 +88,31 @@ public CSharpCompilationOptions CompilationOptions
                 {
                     throw new ArgumentNullException(nameof(value));
                 }
+
                 _compilationOptions = value;
             }
         }
 
+        /// <summary>
+        /// Gets the composite result of <see cref="FileProviders"/>.
+        /// </summary>
+        /// <returns>The <see cref="IFileProvider"/> instance if exactly one instance is registered, otherwise
+        /// a <see cref="CompositeFileProvider"/> that contains all registered instances.</returns>
+        public IFileProvider GetCompositeFileProvider()
```

Comment:
I'm just asking the question. I think either one is a bit cumbersome. Let me ask @lodejard if he's around today.